### PR TITLE
Fixes txp modals for single outputs

### LINKED ISSUE #3084: Payment proposal details disappearing
When the popup showing a new spend proposal is open and the proposal gets accepted/rejected, details disappear:
image

### PR DESCRIPTION
**Pull request by cmgustavo**
fixes #3084

### CODE REVIEW

File: public/views/modals/tx-details.html [hunk 1]
```diff
@@ -46,7 +37,27 @@ <h4 class="title m0" translate>Details</h4>
         <span class="enable_text_select"> {{btx.labelTo || btx.addressTo}}</span>
       </span>
     </span>
+    </li> 
+
+    <li ng-show="btx.hasMultiplesOutputs" class="line-b p10 oh"
```
Comment (greggzigler) [inline public/views/modals/tx-details.html]: when i click on the arrow, it does not expand the list of recipients. perhaps the list is not populated?

File: src/js/services/utilService.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+'use strict';
+
```
Comment (greggzigler) [inline src/js/services/utilService.js]: very nice. less redundancy, and the overall code is cleaner.  good design.

File: public/views/includes/output.html [hunk 3]
```diff
@@ -1,24 +1,14 @@
...
-    <span class="right">{{output.amountStr}}
-      <span ng-show="output.alternativeAmountStr" class="label gray radius">{{output.alternativeAmountStr}}</span>
-    </span>
-  </li>
-  <li class="line-b p10 oh">
-    <span class="text-gray" translate>Note</span>:
-    <span class="right">{{output.message}}</span>
-  </li>
-</div>
+<li class="p10 oh" ng-click="copyAddress(output.toAddress)">
```
Comment (greggzigler) [inline public/views/includes/output.html]: removing the lines between list items is a good UX improvement. much easier for the user to read the list of recipients.

File: src/js/services/utilService.js (hunk 2 above)
Comment (cmgustavo) [inline src/js/services/utilService.js]: Yes, that 's the idea, because right now we haven't any common service of utilities. Maybe we need it later.

**Comment by greggzigler** [comment]
all the functionality works for single and multiple outputs, the user experience is better, and the code is cleaner. nice job!

**Pull request merged**